.sbtopts

```diff
@@ -0,0 +1,3 @@
+-J-Xmx3500M
+-J-Xss2M
+-Dsbt.task.timings=true
```

Comment:
do we need this?

Reply:
It's not mandatory, although there is a similar one in Tapir. Should I remove it?

Reply:
it's just that sbt.task.timings=true is noisy

Reply:
OK, I remove the file (additionally it would clash with the JAVA_OPTS env setting in the current ci.yml)

Reply:
Done